src/ontology/uberon-edit.obo

```diff
@@ -227394,6 +227418,15 @@ is_a: UBERON:0001744 ! lymphoid tissue
 relationship: part_of UBERON:0010417 ! lymph node T cell domain
 created_by: http://orcid.org/0000-0002-2825-0621
 
+[Term]
+id: UBERON:8420000
+name: hair of scalp
```

Comment:
minor: the name sounds like it refers to a complete head of hair rather than a strand of hair

Reply:
I didn't make this change.